docs/docs/components/experimental.mdx

```diff
@@ -170,25 +145,14 @@ To use this component, provide the input value, specify the flow name to run, an
 
 ### Runnable Executor
 
-This component executes a runnable.
+Execute a runnable.
```

Comment:
Runnable should be title case as it is a LangChain class. Maybe it should even be LangChain Runnable